framework/src/problems/FEProblemBase.C

```diff
@@ -4686,6 +4686,18 @@ FEProblemBase::bumpVolumeQRuleOrder(Order order, SubdomainID block)
   updateMaxQps();
 }
 
+void
+FEProblemBase::bumpAllQRuleOrder(Order order, SubdomainID block)
+{
+  for (unsigned int tid = 0; tid < libMesh::n_threads(); ++tid)
+    _assembly[tid]->bumpAllQRuleOrder(order, block);
+
+  if (_displaced_problem)
+    _displaced_problem->bumpAllQRuleOrder(order, block);
+
+  updateMaxQps();
```

Comment:
This function is problematic because it does not resize the default values in `Coupleable` and `MaterialPropertyInterface`.